packages/experimental/NativeFontMetrics/src/useFontMetrics.ts

```diff
@@ -1,20 +1,21 @@
 import { useMemo } from 'react';
 import { NativeEventEmitter, Platform } from 'react-native';
 import { useSubscription } from 'use-subscription';
+import { fontMetrics } from './fontMetrics';
 import NativeFontMetrics from './NativeFontMetrics';
 import { ScaleFactors } from './NativeFontMetrics.types';
 
 const eventEmitter = new NativeEventEmitter(NativeFontMetrics as any);
 
-export function useFontMetrics(): ScaleFactors {
+export function useFontMetricsScaleFactors(): ScaleFactors {
   if (Platform.OS !== 'ios') {
     console.warn('NativeFontMetrics is only available on iOS');
     return {};
   }
```

Comment:
I've seen that these warnings to show up a lot on boot for all platforms. Can we verify this doesn't show up on say, macOS? Or better yet, refactor to make sure this is only used on iOS?

Reply:
Our test code doesn't run macOS code, so as this stands it would be on the client to ensure that this is only used on iOS. That being said, it would also be nice to make sure we don't try to access `NativeFontMetrics` on any other platform.

For some strange reason, I find that if we try splitting this into its own useFontMetrics.ios.ts file, the auto-linking from `pod install` fails to pull FRNFontMetrics into the iOS test app. Apparently the macOS test app has auto-linking problems too, so maybe it's just a weird quirk.

Reply:
I want to follow up on the "auto linking doesn't work" part. That feels strange to me because auto linking should only be dependent on ` pod spec` file being available at the root of your package. 

That being said, I did run this change on macOS and didn't see extra warnings pop up so I think so yay?

Reply:
Either way, I think we can get around this by just having this module always return a multiplier of 1 on non-iOS platforms.